lib/ansible/module_utils/network/iosxr/iosxr.py

```diff
@@ -415,7 +416,14 @@ def load_config(module, command_filter, commit=False, replace=False,
         if module._diff:
             diff = get_config_diff(module)
 
-        if commit:
+        if replace:
+            cmd = {'command': 'commit replace',
+                   'prompt': 'This commit will replace or remove the entire running configuration',
+                   'answer': 'yes'}
+            conn.edit_config(cmd)
+            sleep(5)
```

Comment:
Things like this scary me.

Reply:
So do me, usually the commit replace operation takes say like a 1 or 2 second to complete. I am not sure if our command are sent async; assuming they are async, I added a delay here. With a small config file it works just fine without delay too. Maybe I will comment this out for now, with a note stating add a delay if a failure occurs in future (which I hope won't)

Reply:
afaik after sending command to remote host, we wait for response till default command timeout is triggered (30 sec)